scripts/app-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if [ -n "$RDS_INSTANCE_ADDRESS" ]; then
+    echo "Waiting for postgres..."
+
+    while ! nc -z $RDS_INSTANCE_ADDRESS 5432; do
```

Comment:
could we make this 1234 so it aligns with the readme

Reply:
It is the same with the settings, this value will only need to be changed if using a loopback connection locally to target RDS.